src/fdb5/toc/TocPurgeVisitor.cc

```diff
@@ -70,9 +70,7 @@ void TocPurgeVisitor::gatherAuxiliaryURIs() {
 
         // Add auxiliary files to the corresponding set
         eckit::URI uri(store_.type(), eckit::PathName(it.first));
-        for (const auto& auxURI : store_.getAuxiliaryURIs(uri)) {
-            if (!store_.auxiliaryURIExists(auxURI))
```

Comment:
Not sure why this change? Please make sure we don't break purge (at least in the local-local case). A purge PR will follow, but it needs to work in the interim.